frontend/src/pages/Friends.jsx

```diff
@@ -123,16 +170,25 @@ class Friends extends Component {
 	}
 
 	removeFriend(authorObj){
+		const displayName = this.getLocalDisplayName()
+		const url = this.getLocalHost()
 		const urlPath = "/api/unfollow/"
+		console.log(authorObj)
```

Comment:
remove this console.log or face @shchau 's wrath

Reply:
![e6d32edca3969ea7d282a04cf01992a6](https://user-images.githubusercontent.com/31778426/55020257-179b2400-4fbc-11e9-9d01-e2df4e7efce5.jpg)